.github/workflows/rebase-release-prs.yml

```
@@ -1,8 +1,8 @@
 name: Rebase Release PRs
 on:
-  push:
-    branches:
-    - master
+  workflow_dispatch:
+  schedule:
+    - cron: 0 5 * * MON-FRI
```

Hmm, according to the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), the schedule can be defined using a [POSIX cron schedule](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07).
I think this standard does not contain weekday names, but only the index (0-6, where 0 is Sunday).
So I think you would have to change this to:
```suggestion
    - cron: 0 5 * * 1-5
```

---

Further down in the document it gives an example using SUN-SAT. Our ASF workflow also uses MON. Happy to change it if you prefer it, but it would surprise me if it was an issue.

---

Oh, no, not at all! If it works, this is the cleaner / more readable way of defining the schedule! 👍🏽